### PR TITLE
Pr 4 5 sanda

### PR DESCRIPTION
**Pull request by SandaPanteleimonov**
Added few new functions for districts of our megapolis.

### CODE REVIEW

File: public/sanda/practice/sanda4.5.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        /* List of districts */
+        function getTheBestDistrict() {
+            let districts = [
+                'Ciocana',
+                'Rascani',
+                'Centru',
+                'Buiucani',
+                'Botanica'
+            ];
+            return districts[generateRandomIntInRange(0, 5)];
```
Comment (eclimov) [inline public/sanda/practice/sanda4.5.html]: Please, use length of your `districts` variable instead of `5` here.

**Review by tzatushevskaya — APPROVED** [review]
fine by me

File: public/sanda/practice/sanda4.5.html (hunk 1 above)
Comment (SandaPanteleimonov) [reply public/sanda/practice/sanda4.5.html]: Please, provide more details, because I want to end this quarantine with pull request approved.

Comment (eclimov) [reply public/sanda/practice/sanda4.5.html]: "5" is a hardcoded value here. In this case it means length of your array. But if you add one more item to the array, the code will stop working correctly until you change "5" to "6" manually. This is not flexible.
Knowing how to get length of an array in Javascript, please, avoid hardcode.

File: public/sanda/practice/sanda4.5.html [hunk 2]
```diff
@@ -53,7 +53,7 @@
                 'Buiucani',
                 'Botanica'
             ];
-            return districts[generateRandomIntInRange(0, 5)];
+            return districts[generateRandomIntInRange(0, length)];
```
Comment (eclimov) [inline public/sanda/practice/sanda4.5.html]: This doesn't work correctly.
`length` is a property of array, but you use it as a variable here.
You need to get length of your `districts` variable.

Please, check the example and fix:
https://www.w3schools.com/jsref/jsref_length_array.asp

**Review by eclimov — APPROVED** [review]
Well done!
Approved.

**Pull request merged**